docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,53 @@
+Hello Akka Scala
+================
+
```

Comment:
This Guide will be read from developer.lightbend.com/guides (similar to [http://developer.lightbend.com/guides/hello-akka/](developer.lightbend.com/guides)) so that is the entry point and therefore the guide (this file) should contain instructions of how to create the example project.

Something like this: Use the `akka-scala-seed` in [Get started with Lightbend technologies](http://dev.lightbend.com/start/?group=akka&project=akka-scala-seed)] to create the "Hello World" example project. Unzip the zip file and rename the directory to your preference.